mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule UeberauthTwitter.Mixfile do
     [
      {:httpoison, "~> 0.7"},
      {:oauther, "~> 1.1"},
-     {:poison, "~> 1.3 or ~> 2.0"},
+     {:poison, "~> 1.3 or ~> 2.0 or ~> 3.0 or ~> 4.0"},
```

Comment:
Can we express this any other way?  `"~> 1.3 or > 2.0` perhaps?

Reply:
Hey so that choice re: using `~>1.3 or > 2.0` would be best. Granted, if something breaks in a higher version, it'd be something to report an issue for.